### PR TITLE
Open blocklist

### PR DESCRIPTION
**Pull request by nnsgmsone**
## What type of PR is this?

- [ ] API-change
- [ ] BUG
- [ ] Improvement
- [ ] Documentation
- [ ] Feature
- [ ] Test and CI
- [ ] Code Refactoring

## Which issue(s) this PR fixes:

issue #5924

## What this PR does / why we need it:

### CODE REVIEW

File: pkg/vm/engine/tae/tables/block.go [hunk 1]
```diff
@@ -749,6 +751,7 @@ func (blk *dataBlock) onCheckConflictAndDedup(rowmask *roaring.Bitmap, ts types.
 			blk.mvcc.RLock()
 		}
 		if deleteNode.IsAborted() || !deleteNode.IsVisible(ts) {
+			logutil.Infof("table %d-%v,row %v pk %v", blk.meta.GetSegment().GetTable().ID, blk.meta.GetSchema().Name, row, blk.pkIndex)
```
Comment (XuPeng-SH) [inline pkg/vm/engine/tae/tables/block.go:754]: You can comment on these print by default and enable these print when debugging. It's too noisy.

Comment (nnsgmsone) [reply pkg/vm/engine/tae/tables/block.go:754]: So how should I debug when there is a bug?

Comment (XuPeng-SH) [reply pkg/vm/engine/tae/tables/block.go:754]: Use debug level

File: etc/launch-tae-CN-tae-DN/dn.toml [hunk 2]
```diff
@@ -24,15 +24,15 @@ name = "ETL"
 backend = "DISK-ETL"
 
 [dn]
-uuid = "dd4dccb4-4d3c-41f8-b482-5251dc7a41bf"
+uuid = "dn1"
```
Comment (zhangxu19830126) [inline etc/launch-tae-CN-tae-DN/dn.toml:27]: why change dn's uuid

File: etc/launch-tae-CN-tae-DN/dn.toml [hunk 3]
```diff
@@ -2,13 +2,13 @@ service-type = "DN"
 data-dir = "./mo-data"
 
 [log]
-level = "info"
+level = "error"
```
Comment (zhangxu19830126) [inline etc/launch-tae-CN-tae-DN/dn.toml:5]: no error default

File: etc/launch-tae-CN-tae-DN/cn.toml [hunk 4]
```diff
@@ -28,7 +28,7 @@ disableTrace = true
 disableMetric = true
 
 [cn]
-uuid = "dd1dccb4-4d3c-41f8-b482-5251dc7a41bf"
+uuid = "cn1"
```
Comment (zhangxu19830126) [inline etc/launch-tae-CN-tae-DN/cn.toml:31]: why change cn's uuid

File: etc/launch-tae-CN-tae-DN/cn.toml [hunk 5]
```diff
@@ -2,7 +2,7 @@ service-type = "CN"
 data-dir = "./mo-data"
 
 [log]
-level = "info"
+level = "error"
```
Comment (zhangxu19830126) [inline etc/launch-tae-CN-tae-DN/cn.toml:5]: no error default

**Review by zhangxu19830126 — CHANGES_REQUESTED** [review]

**Review by zhangxu19830126 — APPROVED** [review]

**Review by XuPeng-SH — APPROVED** [review]
lgtm

**Review by yingfeng — APPROVED** [review]

**Review by sukki37 — APPROVED** [review]

**Pull request merged**